### PR TITLE
(Support) Excluded Annotation Types

### PR DESCRIPTION
**Pull request by dcupidon**
Exposed APIs that excluded specified annotation types from the annotation list

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -1245,6 +1245,20 @@ If document editing is enabled, then this value determines if the annotation lis
 />
 ```
 
+#### excludedAnnotationListTypes
+array of string, optional, defaults to none
```
Comment (sgong-pdftron) [inline API.md:1249]: - [ ] array of `Config.Tools`

File: android/src/main/java/com/pdftron/reactnative/views/DocumentView.java [hunk 2]
```diff
@@ -529,6 +529,16 @@ public void setUserBookmarksListEditingEnabled(boolean userBookmarksListEditingE
         mBuilder = mBuilder.userBookmarksListEditingEnabled(userBookmarksListEditingEnabled);
     }
 
+    public void setExcludedAnnotationListTypes(ReadableArray excludedTypes) {
+        int [] annotTypes = new int [excludedTypes.size()];
```
Comment (sgong-pdftron) [inline android/src/main/java/com/pdftron/reactnative/views/DocumentView.java:533]: - [ ] formatting

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

**Comment by sgong-pdftron** [comment]
please update with master

File: ios/RNTPTDocumentView.h [hunk 3]
```diff
@@ -430,6 +430,8 @@ static const PTAnnotationToolbarKey PTAnnotationToolbarKeyItems = @"items";
 
 @property (nonatomic, assign) BOOL saveStateEnabled;
 
+@property (nonatomic, copy, nullable) NSArray<NSString *> *excludedAnnotationListTypes;
```
Comment (JamesTron) [inline ios/RNTPTDocumentView.h:433]:
```suggestion
@property (nonatomic, strong, nullable) NSArray<NSString *> *excludedAnnotationListTypes;
```
All of the `NSArray<NSString *>` should have a storage type of `strong` not `copy`.

File: ios/RNTPTDocumentView.m [hunk 4]
```diff
@@ -571,6 +571,30 @@ - (void)disableElementsInternal:(NSArray<NSString*> *)disabledElements documentV
     [self setToolsPermission:disabledElements toValue:NO documentViewController:documentViewController];
 }
 
+
+- (void)setExcludedAnnotationListTypes:(NSArray<NSString *> *)excludedAnnotationListTypes
+{
+    _excludedAnnotationListTypes = [excludedAnnotationListTypes copy];
```
Comment (JamesTron) [inline ios/RNTPTDocumentView.m:577]:
```suggestion
    _excludedAnnotationListTypes = excludedAnnotationListTypes;
```

**Review by JamesTron — CHANGES_REQUESTED** [review]

**Review by sgong-pdftron — APPROVED** [review]

**Review by dluco — APPROVED** [review]

**Review by brandenfung2 — APPROVED** [review]

**Pull request merged**